Configure

```diff
@@ -912,11 +920,23 @@ while (@argvcopy)
                         }
                 else    # common if (/^[-+]/), just pass down...
                         {
+                        # Treat %xx as an ASCII code (e.g. replace %20 by a space character).
+                        # This provides a simple way to pass options with arguments separated
+                        # by spaces without quoting (e.g. -opt%20arg translates to -opt arg).
                         $_ =~ s/%([0-9a-f]{1,2})/chr(hex($1))/gei;
                         push @{$useradd{CFLAGS}}, $_;
                         push @{$useradd{CXXFLAGS}}, $_;
                         }
                 }
+        elsif (m|^/|)
+                {
+                # Treat %xx as an ASCII code (e.g. replace %20 by a space character).
+                # This provides a simple way to pass options with arguments separated
+                # by spaces without quoting (e.g. /opt%20arg translates to /opt arg).
+                $_ =~ s/%([0-9a-f]{1,2})/chr(hex($1))/gei;
+                push @{$useradd{CFLAGS}}, $_;
+                push @{$useradd{CXXFLAGS}}, $_;
```

Comment:
Have you noticed that we catch `-L`, `-l`, `-I` and `-D`, and that their values end up elsewhere?  Would you mind doing something similar for `/` variants of the same?

Reply:
Ok, I‘ll look into it.

Reply:
Added support for /D /I /L.  (There is no analogue to -l)

Any more [MSVC compiler options](https://docs.microsoft.com/en-us/cpp/build/reference/compiler-options-listed-by-category?view=vs-2019) which need to be treated specially? In particular, I was wondering whether it is useful to support the [/link](https://docs.microsoft.com/en-us/cpp/build/reference/link-pass-options-to-linker?view=vs-2019) option, which seems to be an anologue of gcc's -Wl, . (But that's rather a task for another pull request.)